### DISCUSSION TITLE
Scores are not being uploaded to https://lazer.ppy.sh/

### DISCUSSION THREAD

**Discussion by henribott** [ppy/osu#23423] [Q&A]
Hello, I've been playing the newest version of Lazer for a while but I've noticed that my friend's scores are going to his profile in Lazer (https://lazer.ppy.sh/) and mine aren't. 
As my last score([Taiko] 89pp Mare Mantis - M2U ) was the last map I played, I am providing the logs.

With this, I've also found that my last scores are from 4 months, and then 2 years ago, which makes me wonder if I've had this problem during all this time. I don't play for pp so this is just an inconvenience, but I'd like to add to the discussion in case someone else is also having this issue.
[database.log](https://github.com/ppy/osu/files/11415869/database.log)
[input.log](https://github.com/ppy/osu/files/11415870/input.log)
[network.log](https://github.com/ppy/osu/files/11415871/network.log)
[performance.log](https://github.com/ppy/osu/files/11415872/performance.log)
[runtime.log](https://github.com/ppy/osu/files/11415873/runtime.log)
[updater.log](https://github.com/ppy/osu/files/11415874/updater.log)

**Comment by peppy** [comment]
Did you attempt sending a score in the logs above? I can't see any attempt on an initial inspection. Can you re-attach logs after trying?

**Comment by henribott** [comment]


**Reply by Theighlin** [reply]
No need to, multiplayer scores are not being uploaded yet.
